flake.nix

```
@@ -1,5 +1,5 @@
 {
-  description = "High-level tracing language for Linux eBPF";
+  description = "High-level tracing language for Linux";
```

Nit: we could add BPF reference here (and also to GitHub project summary) but no strong opinion about this:
```suggestion
  description = "High-level tracing language for Linux powered by eBPF";
```

---

Let's keep in line w/ GH project about for now (Alastair changed yesterday). We;re basically debating mission statement (useful) and this would be good to get more consensus on